### ISSUE TITLE
Data Science: Prediction

### ISSUE THREAD

**Issue by europe-asia-america** [GeekHaven/HacktoberFest-AI#26]
Create a simple predictor of the stock prices using an LSTM.

1. Select a previously created dataset from #6.
2. Use that dataset to train your LSTM by taking a certain section of it as your training set (preferably the first 80% of the dataset chronologically speaking).
3. Generate continuation data that is to be compared with the test set (the rest of the original dataset that you haven't used in training the LSTM).
3. Use your code for #7 to visualize the difference between the predictions and the actual outcomes.

**Comment by julienneross** [comment]
Does any of you have or a link to any data prediction that is already program? I would love to study it. Thanks!
